navel.lisp

```diff
@@ -126,6 +126,36 @@
 ;;         (multiple-value-call 'format stream "~&~@{~A~#[~:; ~]~}~%" name
 ;;           (trades-profits day))))))
 
+;; this runs on (slot-reduce maker lictor trades) or CSV tradelog
+;; (let (skipped (trades (sort * '> :key 'volume)))
```

Comment:
at a high level, the cost basis calculation is quite similar to this one, although there the relevant data gets lost and only the net effect of exclusions get summed... perhaps an improvement with linear complexity would run through one time doing the cost basis reduction, and then pick exclusions deterministically based on that result?